observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The {coo-full}
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+You can use the {coo-first} to install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
+
+["dashboards_{context}"]
+== Dashboards
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus data sources from the cluster to the default dashboards, in addition to the in-cluster data source.
+This results in a unified observability experience across different data sources.
+
+["troubleshooting_{context}"]
+== Troubleshooting
+
+The troubleshooting panel UI plugin provides observability signal correlation, powered by the open source Korrel8r project.
+You can use the troubleshooting panel available from the *Observe* -> *Alerting* page to easily correlate metrics, logs, alerts, netflows, and additional observability signals and resources, across different data stores.
+The output of Korrel8r is displayed as an interactive node graph. When you click on a node, you are automatically redirected to the corresponding web console page with the specific information for that node, for example, metric, log, or pod.
+
+["distributed-tracing_{context}"]
+== Distributed tracing
+
+The distributed tracing UI plugin adds tracing-related features to the web console on the *Observe* -> *Traces* page.
+You can follow requests through the front end and into the backend of microservices, helping you identify code errors and performance bottlenecks in distributed systems.
+You can select a supported `TempoStack` or `TempoMonolithic` instance running in the cluster and set a time range and query to view the trace data.
```

Comment:
We only support `TempoStack` with multi-tenancy enabled, other types like `TempoMonolithic` or `TempoStacks` without multi-tenancy are not shown

Reply:
Done